.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	extends: [ 'plugin:@woocommerce/eslint-plugin/recommended' ],
+	globals: {
+		_: false,
+		Backbone: false,
+		jQuery: false,
+		wp: false,
+	},
+	settings: {
+		jsdoc: { mode: 'typescript' },
```

Comment:
I'm assuming the team managing PayFast is okay with typescript flavored jsdoc, but should be confirmed.